src/bindings/python/src/openvino/frontend/pytorch/patch_builtins.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import builtins
+import torch
+from typing import Callable, Dict
+
+originals_map = {
+    'divmod': builtins.divmod
+}
+
+
+def patched_divmod(a, b):
+    if isinstance(a, torch.Tensor):
+        q = torch.div(a, b, rounding_mode='trunc')
+        r = torch.remainder(a, b)
+        return q, r
+    return originals_map['divmod'](a, b)
+
+
+patches_map = {
+    'divmod': patched_divmod,
+}
+
+
+class BuiltinPatcher:
```

Comment:
This can be used not only to patch built-ins, but any function, so I recommend renaming it

Reply:
Any ideas for naming?

Reply:
`FunctionPatcher`, although I don't like it, it is better then the current one

Reply:
Done